renovate.json

```
@@ -1,5 +1,6 @@
 {
   "extends": ["github>freecodecamp/renovate-config"],
+  "schedule": ["after 5pm on Monday"],
```

The `schedule` option only determines when new branches are *created*. It won't do anything for existing branches.

We likely want:

```suggestion
  "rebaseWhen": "conflicted"
```

Which will change the behaviour for *updating* branches to only do so if a conflict is detected, rather than every time the branch is out of date.

---

We may wish to discuss this further, first - Renovate's documentation advises against setting this while using auto-merge.

---

Yeah, with good reason: we don't want to rely on git to determine if a lockfile is valid or not.

---

There's also the risk that a change made in a merged PR works with the existing package, but breaks in an updated version - we wouldn't catch that, because the version bump PR wouldn't get updated.

---

Alright, well, what's the worth of auto merge if we only allow two pull-requests at a time to be open? It wouldn't be much effort to manually approve it anyway.

---

We say that, but keeping up on Renovate PRs is kind of a nightmare.

---

> There's also the risk that a change made in a merged PR works with the existing package, but breaks in an updated version - we wouldn't catch that, because the version bump PR wouldn't get updated. 

Why not rebase it first before merging. I know it's a stretch.

---

Because past experience has shown us that we just simply don't have the capacity to keep up on all of the Renovate PRs manually.